src/components/TradeAssetSearch/components/SearchTermAssetList.tsx

```diff
@@ -35,14 +37,20 @@ export const SearchTermAssetList = ({
   const assets = useAppSelector(selectAssetsSortedByName)
   const assetsById = useAppSelector(selectAssets)
   const walletConnectedChainIds = useAppSelector(selectWalletConnectedChainIds)
+  const fetchAndUpsertCustomTokenMarketData = useFetchAndUpsertCustomTokenMarketData()
   const chainIds = useMemo(
     () => (activeChainId === 'All' ? walletConnectedChainIds : [activeChainId]),
     [activeChainId, walletConnectedChainIds],
   )
   const walletSupportedEvmChainIds = useMemo(() => chainIds.filter(isEvmChainId), [chainIds])
+  const alchemySupportedChainIds = useMemo(
```

Comment:
a bit more accurate vernacular since it is the subset of wallet supported and alchemy supported chain ids for custom token `chainIds`
```suggestion
  const customTokenSupportedChainIds = useMemo(
```

Reply:
https://github.com/shapeshift/web/pull/7370/commits/9f7c5f1860557b4f526baab4c81aa1cdd6029341